src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -465,6 +465,24 @@ pub enum CreateConnector<T: AstInfo> {
     },
 }
 
+impl<T: AstInfo> AstDisplay for CreateConnector<T> {
+    fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
+        match self {
+            Self::Kafka {
+                broker,
+                with_options,
+            } => {
+                f.write_str("KAFKA BROKER '");
+                f.write_node(&display::escape_single_quote_string(broker));
```

Comment:
Does this need to check that `!with_options.is_empty()`? Or are the options always required?

Reply:
They should be required and I will update the parser to reflect that.